docs/pages/includes/commercial-prereqs-tabs.mdx

```diff
@@ -1,13 +1,19 @@
-<Tabs>
-<TabItem
-  scope={["enterprise"]} label="Self-Hosted Enterprise">
-
-- A running Teleport Enterprise cluster. For details on how to set this up, see the Enterprise [Getting 
-Started](../choose-an-edition/teleport-enterprise/introduction.mdx) guide.
+- Access to one of the commercial editions of Teleport running in your environment:
+  
+  - Teleport Enterprise
+  - Teleport Team 
```

Comment:
We have to be careful here. This partial is used on pages that refer to features that are not available in Teleport Team, like SSO.